runtime/shared_common/shrinit.cpp

```diff
@@ -4914,16 +4914,18 @@ adjustCacheSizes(J9PortLibrary* portlib, UDATA verboseFlags, J9SharedClassPreini
 void
 j9shr_jvmPhaseChange(J9VMThread *currentThread, UDATA phase)
 {
-	if (J9VM_PHASE_NOT_STARTUP == phase) {
-		J9JavaVM* vm = currentThread->javaVM;
+	J9JavaVM* vm = currentThread->javaVM;
 
+	if (J9VM_PHASE_NOT_STARTUP == phase) {
```

Comment:
I think we should add a tracepoint to this function as well, that prints the new phase and the old phase.

Reply:
Sorry, I mean that comment for jvmPhaseChange